release-notes/azure/v11.2.2.md

```diff
@@ -0,0 +1,26 @@
+## ⚡️ Giant Swarm Release 11.2.2 for Azure ⚡️
+
+This release contains improvements in the upgrade procedure.
+We have improved new nodes discovery at the stage where the cluster size is doubled with new versions of the machines.
+We experienced issues with stuck upgrades where not enough quota was available, breaking the state of azure-operator at new nodes discovery. Thanks to improvements no manual action will be needed from Giant Swarm side to recontinue the upgrade when the quotas are met again.
+
+The release also includes couple of reliability improvements to pre-installed apps such as NGINX Ingress Controller and kube-state-metrics.
```

Comment:
```suggestion
The release also includes reliability improvements to these pre-installed apps: NGINX Ingress Controller and kube-state-metrics.
```

Just made it shorter. And made it clear only two apps were changed. Instead of implying that there are more.